releasenotes/notes/better_mcx_synthesis-7e6e265147bc1d33.yaml

```
@@ -0,0 +1,16 @@
+---
+features_synthesis:
+  - |
+    Added new MCX gate decompositions utilising conditionally clean ancillae, improving circuit
+    depth and efficiency. New synthesis methods `synth_mcx_1_clean_kg24`, `synth_mcx_`_dirty_kg24`,
+    `synth_mcx_2_clean_kg24`, `synth_mcx_2_dirty_kg24` have been added to
+    `qiskit/synthesis/multi_controlled/mcx_synthesis.py`.
```

A few nitpicks about the release notes. It's recommended to link the referenced gates classes and functions. US English is preferred (so "utilizing" instead of "utilising").  There is a typo in ```synth_mcx_`_dirty_kg24```. We also don't reference the actual code (i.e. best not to mention ``qiskit/synthesis/multi_controlled/mcx_synthesis.py``). You may also want to briefly highlight the difference between the added methods.

I have provided a suggestion, but I have not actually tried it out, and you are more than welcome not to follow it.

```suggestion
    Added new decompositions for :class:`.MCXGate` utilizing clean ancillae, improving circuit
    depth and efficiency:
    
    * :func:`.synth_mcx_1_clean_kg24`, using 1 additional clean ancilla qubit
    * :func:`.synth_mcx_1_dirty_kg24`, using 1 additional dirty ancilla qubit
    * :func:`.synth_mcx_2_clean_kg24`, using 2 additional clean ancillary qubits
    * :func:`.synth_mcx_2_dirty_kg24`, using 2 additional dirty ancillary qubits